docs/contributors/opentracingprovider.adoc

```diff
@@ -177,6 +177,18 @@ The value should be used to set the componentType field on a `Component` node.
 
 If the transaction has not been defined in the trace context, then the value of this tag will be used to set it. This value will then be propagated to other subsequent trace contexts.
 
+==== Tag ‘service’
+
+A free-format value that ties this request to a specific service. When the application is executed in OpenShift, this defaults to a value that is derived from the environment variable `OPENSHIFT_BUILD_NAME`,
+prefixed with the namespace specified via the environment variable `OPENSHIFT_BUILD_NAMESPACE`. For instance, if the `OPENSHIFT_BUILD_NAME` is `bar-1` and the `OPENSHIFT_BUILD_NAMESPACE` is `foo`, this
+value is `foo.bar`
```

Comment:
Need to make it clear that the service name is independent of the specific version number?